### PR TITLE
[CONTACT]: model - migration - seeder - OT102-71

### PR DESCRIPTION
**Pull request by gabyMangoni**
[OT102-71](https://alkemy-labs.atlassian.net/browse/OT102-71)
-----------------------------------------------------------------------------------------------------------------------------------------

**Description:**
-----------------------------------------------------------------------------------------------------------------------------------------
AS: administrator user
I WANT: Store the data when completing the Contact Form
TO: Have a Contacts database

**Acceptance criteria:**
-----------------------------------------------------------------------------------------------------------------------------------------
By completing the Contact Form, it will be stored in the database. Table name: contacts. It will contain the fields name, phone, email, message, deletedAt (used for soft delete)

**How to reproduce this change:**
-----------------------------------------------------------------------------------------------------------------------------------------
apply migration:
npx sequelize-cli db:migrate

undo migration:
npx sequelize-cli db:migrate:undo:all --to XXXXXXXXXXXXXX-nombre-migracion.js

### CODE REVIEW

File: migrations/20211122231452-create-contact.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      name: {
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
+      phone: {
+        type: Sequelize.INTEGER,
```
Comment (dvelezroman) [inline migrations/20211122231452-create-contact.js]: phone debe ser un STRING

**Review by dvelezroman — CHANGES_REQUESTED** [review]
Solo un pequeno cambio en el tipo de phone

File: models/contact.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    static associate(models) {
+      // define association here
+    }
+  };
+  Contact.init({
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    phone: DataTypes.INTEGER,
```
Comment (dvelezroman) [inline models/contact.js]: phone  debe ser STRING

**Review by dvelezroman — APPROVED** [review]
Excelente

**Review by JonatanGarbuyo — APPROVED** [review]

**Review by gabodmc — APPROVED** [review]

**Review by alexiscjscab — APPROVED** [review]

**Pull request merged**